keyboards/zlant/keymaps/default/keymap.c

```diff
@@ -20,3 +20,27 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   )
 
 };
+
+
+/*  Base
+ * ,-----------------------------------------------------------------------------------.
+ * | TAB  |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Bksp |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * | LCTRL|   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |  "   |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * | ESC  |   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Enter |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |LED H+|LED H-| GUI  | Alt  | SHIFT|    Space    |Raise | Left | Down |  Up  |Right |
+ * `-----------------------------------------------------------------------------------'
+ *
+ *  FN
+ * ,-----------------------------------------------------------------------------------.
+ * |      |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  |      |
+ * |------+------+------+------+------+-------------+------+------+------+------+------|
+ * |      |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |   -  |   =  |   [  |   ]  |  \   |
+ * |------+------+------+------+------+------|------+------+------+------+------+------|
+ * |      |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |      | pgup | Home |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |LED V+|LED V-| Reset| Pscr |      |             |      | pgdn | End  |      | Del  |
+ * `-----------------------------------------------------------------------------------'
+ */
```

Comment:
We tend to prefer having trailing newlines at the bottom of files, so you might want to add an empty line at the end here 😃